README.md

```diff
@@ -51,18 +52,28 @@ For detailed information on how configuration of plugins works, please refer to
 
 ### type
 
-The type of [Key Generator](#key-generators) to be used.
+The type of [Data Generator](#data-generators) to be used.
 
 *Default:* `'file-hash'`
 *Alternatives:* `'git-tag-commit'`, `'version-commit'`
 
-## Key Generators
+## Data Generators
 
-Key generators are the strategies used to generate the unique revision key. Currently there is only one Key Generator implementation but we will add more as needed. Some examples of other potential key generators are `GitHash` or `DateTime` generators.
+Data generators are the strategies used to generate information about the revision being deployed. A data generator must return an object which contains a property called `revisionKey` which uniquely identifies the current revision. A generator can add any other data that it deems relevant to the data object that it returns.
```

Comment:
Yep, will do the README change first.
On Tue, 1 Sep 2015 at 19:16 Luke Melia <notifications@github.com> wrote:

> In README.md
> <https://github.com/zapnito/ember-cli-deploy-revision-key/pull/16#discussion_r38452360>
> :
>
> >
> > -Key generators are the strategies used to generate the unique revision key. Currently there is only one Key Generator implementation but we will add more as needed. Some examples of other potential key generators are `GitHash` or `DateTime` generators.
> > +Data generators are the strategies used to generate information about the revision being deployed. A data generator must return an object which contains a property called `revisionKey` which uniquely identifies the current revision. A generator can add any other data that it deems relevant to the data object that it returns.
>
> Actually, will wait for your README tweak.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zapnito/ember-cli-deploy-revision-key/pull/16/files#r38452360>
> .
>